tests/tests_app/components/database/test_client_server.py

```diff
@@ -160,3 +161,40 @@ def run(self):
     sleep(2)
 
     os.remove(id)
+
+
+@pytest.mark.skipif(sys.platform == "win32", reason="currently not supported for windows.")
+@pytest.mark.skipif(not _is_sqlmodel_available(), reason="sqlmodel is required for this test.")
+def test_work_database_periodic_store():
```

Comment:
We could use tmpdir here so that the test does not create artifacts in the working dir

Reply:
Done